.travis.yml

```diff
@@ -13,5 +13,13 @@ jobs:
       script: yarn build && yarn test
     - stage: release
       script: bin/ci-release.sh
+      deploy:
+        provider: releases
+        skip_cleanup: true
+        file: syntaxes/Scala.tmLanguage.json
```

Comment:
Issue #75 is about the missing `.vsix` file. The `Scala.tmLanguage.json` already included in the sources zip artefact.

Reply:
I've amended the commit to fix this